Antitrust-Policy.md

```diff
@@ -0,0 +1,64 @@
+
+In the event of any conflict or inconsistency between this Policy and Antitrust Laws, the Antitrust Laws shall control.
+
+## Freedom to Operate
+CNOE does not set standards. CNOE publishes Recommended Best Practices based upon technical merit the adoption of which is voluntary. 
+
+Nothing in this Policy, or in any CNOE document or policy, prohibits or limits the ability of Members or Participants to make, sell or use any product or service, or otherwise to compete in the marketplace. 
+Members and Participants shall not, directly or indirectly, prevent or attempt to prevent others from manufacturing, selling, or supplying any product or service as an alternative to any Recommended Best Practice. Further, Members and Participants shall not promote or require standard of commercial terms, including terms for license and sale.
+
+No Member or Participant shall impose any requirement, obligation or penalty for the use, adoption or contribution to the Recommended Best Practices nor provide any incentive, perk or financial benefit for the use, adoption or contribution to the Recommended Best Practices. Nothing in this policy, however, shall prevent any Member or Participant from developing, marketing, or selling any product or service that incorporates, implements, or supports the Recommended Best Practices.
+
+No Member or Participant shall enter into any agreement to prohibit or restrict any Member, Participant or non-member from establishing or adopting any alternative technologies to the Recommended Best Practices.
+
+While Members and Participants may elect to make contribute to the Recommended Best Practices, none of these contributions shall be exclusive to CNOE. Members and Participants shall be free to contribute to any other open source projects of their choosing, including contributions to Recommend Best Practices open source projects. Such contributions are voluntary. No Member or Participant shall require contributions to any open source project or the adoption of any particular open source software license. 
+
+## Membership & Participation
+As part of its governing charter, CNOE shall establish and maintain reasonable requirements for CNOE membership that are consistent with the efficient and effective pursuit of CNOE’s purpose and goals, and all qualifying entities shall be admitted to join. Such membership shall be inclusive and non-discriminatory as to cloud providers, technology providers and customers of each of the foregoing. No qualifying entity shall be denied membership or removed from membership for any anti-competitive purpose. 
+
+Participation in CNOE by non-member Participants shall be broadly open and no participation shall be excluded except for breaches of the CNOE Code of Conduct and this Policy. 
```

Comment:
same as above. based on what is discussed later in the document ""participation is open to all participants consistent with CNOE’s governing charter" and the CNOE governing charter includes the membership tiers document that we have been floating around.